src/main/java/com/bravo/user/service/PaymentService.java

```diff
@@ -0,0 +1,42 @@
+package com.bravo.user.service;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.utility.PageUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+
+@Service
+public class PaymentService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);
```

Comment:
This `Logger` is bound to the wrong class.

Reply:
Ah yes it is, I was modeling this after the User Resource as per the instructions and forgot to change this.